migrations/V162__Update_MDM_To_MDM_POR.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TYPE OPERATIONAL_SOLUTION_KEY RENAME VALUE 'MDM' TO 'MDM-POR';
+
+UPDATE possible_operational_solution
+SET sol_name = 'Master Data Management Program-Organization Relationship',
+    sol_key = 'MDM-POR',

```

Comment:
This line (sol_key = 'MDM-POR',) isn't needed, the alter type expression already renames the key.